Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -78,15 +114,21 @@
     'vtPositives': 10,
     'vtPositiveUrlsForIP': 30
 }
-# The dictionary below does not represent DBot Scores correctly, and should not be used
-dbotscores = {
-    'Critical': 4,
-    'High': 3,
-    'Medium': 2,
-    'Low': 1,
-    'Unknown': 0,
-    'Informational': 0.5
-}
+
+
+class IndicatorType:
+    """
+    Enum: contains all the indicator types, used DBotScore.Type)
+    """
+    IP = 'ip'
+    HASH = 'hash'
+    DOMAIN = 'domain'
+    URL = 'url'
+
+    @staticmethod
+    def is_valid_type(_type):
```

Comment:
Suggests the function below, doesn't require maintenance. also, we should check that the key exists and not the value.
```python
@classmethod
def is_valid_type(cls, _type):
    # type: (str) -> book
    types = [
        var
        for var in cls.__dict__.keys()
        if not var.startswith("__")
    ]
    return _type in types
```

if you still think it should be by value, replace top `var` with ```cls.__dict__[var]```